docs/pages/eas-update/deployment-patterns.mdx

```diff
@@ -83,18 +83,29 @@ This flow is great for managing versioned releases. Here are the parts of the de
 3. Set up `expo-github-action` to publish updates when merging commits to branches.
 4. Merge changes into a branch named "version-1.0".
 5. Use the website or EAS CLI to point the "staging" channel at the EAS Update branch "version-1.0". Test the update by opening the apps on TestFlight and the Play Store Internal Track.
-6. When ready, use the website or EAS CLI to point the "production" channel at the EAS Update branch "version-1.0".
-7. Then, create another GitHub branch named "version-1.1", and merge commits until the new features and fixes are ready. When they are, use the website or EAS CLI to point the "staging" channel at the EAS Update branch "version-1.1".
-8. When ready, use the website or EAS CLI to point the "production" channel at the EAS Update branch "version-1.1".
+6. When ready, use the website or EAS CLI to point the "production-rtv-1" channel at the EAS Update branch "version-1.0".
+7. Then, there are two update scenarios you may encounter:
+    - A new release does not require a new runtime version:
+        1. Create another GitHub branch named "version-1.1".
+        2. Use the website or EAS CLI to point the "staging" channel at the EAS Update branch "version-1.1".
+        3. Merge commits into the "version-1.1" branch until the new features and fixes are ready and stable.
+        4. Use the website or EAS CLI to point the "production-rtv-1" channel at the EAS Update branch "version-1.1". This will mean that everyone with a production build (users who downloaded the app from the app stores) will now get the latest update on the "version-1.1" branch.
+    - A new release requires a new runtime version (for example, when new native libraries are added or SDK version is upgraded):
+        1. Create a new "staging" build with the new runtime version.
+        1. Create another GitHub branch named "version-1.1".
+        2. Use the website or EAS CLI to point the "staging" channel at the EAS Update branch "version-1.1".
+        3. Merge commits into the "version-1.1" branch until the new features and fixes are ready and stable.
+        4. Create a new build with channel named "production-rtv-2", which will eventually get reviewed and become available on app stores.
+        5. Use the website or EAS CLI to point the "production-rtv-2" channel at the EAS Update branch "version-1.2". This will mean that everyone who previously had a production build (users who downloaded the app from the app stores) will continue to get the latest update on EAS Update branch "version-1.1" until they download the new version of the app from the app store, at which time they will get the latest update on EAS Update branch "version-1.2".
```

Comment:
```suggestion
        5. Use the website or EAS CLI to point the "production-rtv-2" channel at the EAS Update branch "version-2.0". This will mean that everyone who previously had a production build (users who downloaded the app from the app stores) will continue to get the latest update on EAS Update branch "version-1.1" until they download the new version of the app from the app store, at which time they will get the latest update on EAS Update branch "version-2.0".
```